tests/integration/backup_tests/test_backups.py

```diff
@@ -71,20 +72,31 @@ async def add_writes_to_db(ops_test: OpsTest):
     await clear_writes_action.wait()
 
 
+@pytest.mark.skipif(
+    os.environ.get("PYTEST_SKIP_DEPLOY", False),
+    reason="skipping deploy, model expected to be provided.",
+)
```

Comment:
this can be removed - we don't want to skip if this is provided, because we don't know if they have the correct number of units, which is the purpose of `check_or_scale_app`

Reply:
Done